guide/glossary.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: guide
+title: Glossary
+nav_order: 4
+---
+
+# Glossary
+{: .no_toc }
+
+## Address
+
+An identifier of 26-35 alphanumeric characters that can be used to receive bitcoin. There are [several address formats](https://en.bitcoin.it/wiki/List_of_address_prefixes){:target="_blank"} based on different standards and use cases. Applications should make it clear to users which standards are being used. Support for formats varies, so users may need to know this information for backup and future recovery.
```

Comment:
```suggestion
An identifier of 26-35 alphanumeric characters that is used to receive bitcoin. There are [several address formats](https://en.bitcoin.it/wiki/List_of_address_prefixes){:target="_blank"} based on different specifications. Applications should inform users which format it uses as support in varies across applications. Users need to know this information during backup for future recovery.
```

NOTE: In some places we say "wallet-products" and here we say "application". So we should have a conversation on what this should be called to have more harmony in the content - as I've also been leaning towards "applications", "bitcoin applications", "experiences" which introduces more inconsistencies.

Reply:
I am changing product --> application where I encounter it